modin/engines/dask/pandas_on_dask/frame/partition.py

```diff
@@ -351,3 +346,52 @@ def empty(cls):
             A new ``PandasOnDaskFramePartition`` object.
         """
         return cls(pandas.DataFrame(), 0, 0)
+
+
+def apply_func(partition, func, *args, **kwargs):
```

Comment:
Can we merge this with https://github.com/modin-project/modin/blob/ccb5a537d5b776c0c5015f1d3dcf13886d21a250/modin/engines/dask/task_wrapper.py#L23-L46?

If we do the same for Ray, we can reduce the size of these files because it will be standardized.

Reply:
I don't think we can merge this exactly with RayTask.deploy because the latter is applicable for deploying io function. If we really want to merge such things, I would suggest to make the following structure:
```
.task_wrapper/
   .io_task.py # RayIOTask.deploy with own parameters for deploying function
   .partition_task.py # RayPartitionTask.deploy with own parameters for deploying function
   .axis_partition_task.py # RayAxisPartitionTask.deploy with own parameters for deploying function
   . # and probably we will need something like two_axis_partition_task.py containing RayTwoAxisPartitionTask.deploy with own parameters for deploying function

# or we can do the following
# task_wrapper.py
class RayTask:
    def deploy_io_task(...):
        ...
    def deploy_partition_task(...):
        ...
    def deploy_axis_partition_task(...):
        ...
    def deploy_two_axis_partition_task(...):
        ...
```
And then add a class member of RayTask to Partition, AxisPartition and IO classes to call deploy_... functions.
But I would suggest doing this as part of #2957 or after we define a stable Modin structure. What do you think?

Reply:
@YarShev Yes, that makes sense. We can do it later.

Reply:
Cool!